src/core/layout/qgslayoutitempicture.h

```diff
@@ -229,11 +229,20 @@ class CORE_EXPORT QgsLayoutItemPicture: public QgsLayoutItem
     void setSvgStrokeWidth( double width );
 
     /**
-     * Returns the current picture mode (image format).
-     * \see setMode()
+     * Returns the current picture mode (image format), FormatUnknown if given
+     * picture format is unknown
+     * \see setMode() originalMode()
      */
     Format mode() const { return mMode; }
 
+    /**
+     * Returns the original set picture mode (image format).
+     * It could differ from mode() if given picture format is unknown
+     * \see setMode() mode()
+     * \since 3.22
```

Comment:
Shouldn't it be 3.28 here ?
And change since in backport PR ?

Reply:
Not completely sure, but I would say that the since need to be consistent (the same) between version to be sure when the API has changed. And since I plan to backport, the API will have changed in 3.22.